docs/en/manuals/live-update-scripting.md

```diff
@@ -0,0 +1,128 @@
+
+```lua
+
+-- You'll need to track which archive contains which content
+-- In this example, we only use a single liveupdate archive, containing all missing resource.
+-- If you are using multiple archive, you need to structure the downloads accordingly
+local lu_infos = {
+    liveupdate = {
+        name = "liveupdate",
+        priority = 10,
+    }
+}
+
+local function get_lu_info_for_level(level_name)
+    if level_name == "level1" then
+        return lu_infos['liveupdate']
+    end
+end
+
+local function mount_zip(self, name, priority, path, callback)
+	liveupdate.add_mount(name, "zip:" .. path, priority, function(_uri, _path, _status) -- <1>
+		callback(_uri, _path, _status)
+	end)
+end
+
+function init(self)
+    self.http_url = sys.get_config_string("game.http_url", nil) -- <2>
+
+    local level_name = "level1"
+
+    local info = get_lu_archive_for_level(level_name) -- <3>
+
+    msg.post("#", "load_level", {level = "level1", info = info }) -- <4>
+end
+
+function on_message(self, message_id, message, sender)
+    if message_id == hash("load_level") then
+        local missing_resources = collectionproxy.missing_resources("#" .. message.level) -- <5>
+
+        if #missing_resources then
+            msg.post("#", "download_archive", message) -- <6>
+        else
+            msg.post("#" .. message.level, "load")
+        end
+
+    elseif message_id == hash("download_archive") then
+		local zip_filename = message.info.name .. ".zip"
+		local download_path = sys.get_save_file("mygame", zip_filename)
+        local url = self.http_url .. "/" .. zip_filename
+
+        -- Make the request. You can use credentials
+        http.request(url, "GET", function(self, id, response) -- <7>
+			if response.status == 200 or response.status == 304 then
+				mount_zip(self, message.info.name, message.info.priority, download_path, function(uri, path, status) -- <8>
+					msg.post("#", "load_level", message) -- try to load the level again
+				end)
+
+			else
+				print("Failed to download archive ", download_path, "from", url, ":", get_status_string(status))
+			end
+		end, nil, nil, {path=download_path})
+
+    elseif message_id == hash("proxy_loaded") then -- the level is loaded, and we can enable it
+        msg.post(sender, "init")
+        msg.post(sender, "enable")
+    end
+end
+```
+
+1. The `liveupdate.add_mount()` mounts a single archive using a specified name, priority and a zip file.
+2. You need to store the archive online (e.g. on S3), where you can download it from.
+3. Given a collection proxy name, you need to figure our which archive(s) to download, and how to mount them
+4. At startup, we try to load the level.
+5. Check if the collection proxy has all resources available.
+6. If there are resources missing, then we need to download the archive and mount it.
+7. Make a http request and download the archive to `download_path`
+8. The data is downloaded, and it's time to mount it to the running engine.
```

Comment:
Maybe we should add info that data is available to use right after mounting?
Because next sentence makes me think that we have to restart the engine.